charts/finrem-ns/values.yaml

```diff
@@ -1,15 +1,10 @@
 java:
+  aadIdentityName: finrem
+  ingressHost: "finrem-ns-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal"
```

Comment:
sorry i missed this earlier your chart version also needs to be updated to use this feature:
https://github.com/hmcts/finrem-notification-service/blob/master/charts/finrem-ns/requirements.yaml#L3

https://github.com/hmcts/chart-java/releases (2.9.1 currently)

Reply:
bumped to 2.9.1